lms/djangoapps/teams/static/teams/js/views/teams.js

```diff
@@ -0,0 +1,26 @@
+;(function (define) {
+    'use strict';
+    define([
+        'teams/js/views/team_card',
+        'common/js/components/views/paginated_view'
+    ], function (TeamCardView, PaginatedView) {
+        var TeamsView = PaginatedView.extend({
+            type: 'teams',
+            cardView: TeamCardView,
+
+            events: {
+                'click button.action': '' // entry point for team creation
+            },
+
+            render: function () {
+                PaginatedView.prototype.render.call(this);
+
+                this.$el.append(
+                    $('<button class="action action-primary">' + gettext('Create new team') + '</button>')
```

Comment:
@dan-f @explorerleslie per our conversation, here's my recommendation:
"Can't find a team? You can [view all topics](#) or [search for teams within a topic](#). Try to join an existing team before you [add a team](#)."